stdlib/map.gr

```diff
@@ -280,6 +280,8 @@ let setInArray = array => {
     array[i] = (key, value)
     // no decRef on key and value, since they are stored in array
     Memory.decRef(WasmI32.fromGrain(iter))
+    Memory.incRef(WasmI32.fromGrain((+)))
```

Comment:
Will be a conflict with #1074

Reply:
Nah because this is still @disableGC code.

Reply:
Ah, I guess this hasn't been migrated in a follow-on PR yet. Carry on